db/schema.rb

```diff
@@ -251,5 +251,15 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "webauthn_configurations", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.string "name", null: false
+    t.text "public_key", null: false
```

Comment:
This is fine as a text field. We need to store slightly more than just the public key, so making this a `credential` field rather than just a `public_key` field that can store a serialized hash will be generic enough to support the structure as described in https://github.com/cedarcode/webauthn-ruby#authentication

Reply:
What other information would be required besides public key?  I believe the credential id is actually the user id.